spec/features/user_sees_doomsday_flash_link_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature "doomsday flash message is shown" do
+  scenario "user who has spent greater than $500 is shown doomsday flash message" do
+    travesty = create_list(:travesty_with_items, 4)
+    user = create(:user)
+    ApplicationController.any_instance.stub(:current_user).and_return(user)
+
+    visit items_path
+    first(:button, 'Add to cart').click
+    first(:button, 'Add to cart').click
+    first(:button, 'Add to cart').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.